apps/OboeTester/app/src/main/java/com/mobileer/oboetester/AnalyzerActivity.java

```diff
@@ -78,27 +79,27 @@ protected String getCommonTestReport() {
         report.append("build.fingerprint = " + Build.FINGERPRINT + "\n");
         try {
             PackageInfo pinfo = getPackageManager().getPackageInfo(getPackageName(), 0);
-            report.append(String.format("test.version = %s\n", pinfo.versionName));
-            report.append(String.format("test.version.code = %d\n", pinfo.versionCode));
+            report.append(String.format(Locale.getDefault(), "test.version = %s\n", pinfo.versionName));
```

Comment:
You don't need to pass Locale.getDefault(). That is the default.
https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#format(java.lang.String,%20java.lang.Object...)

Reply:
You get the warning `
Implicitly using the default locale is a common source of bugs: Use String.format(Locale, ...) instead` if you don't do this

Reply:
Updated to Locale.US